src/grid/ZoneGridView.php

```diff
@@ -0,0 +1,137 @@
+<?php
+/**
+ * Domain plugin for HiPanel
+ *
+ * @link      https://github.com/hiqdev/hipanel-module-domain
+ * @package   hipanel-module-domain
+ * @license   BSD-3-Clause
+ * @copyright Copyright (c) 2015-2017, HiQDev (http://hiqdev.com/)
+ */
+
+namespace hipanel\modules\domain\grid;
+
+use hipanel\grid\BoxedGridView;
+use hipanel\grid\RefColumn;
+use hipanel\modules\domain\models\Zone;
+use hipanel\modules\domain\widgets\combo\RegistryCombo;
+use hipanel\widgets\IconStateLabel;
+use Yii;
+use yii\bootstrap\Html;
+use yii\grid\DataColumn;
+
+class ZoneGridView extends BoxedGridView
+{
+    public function columns()
+    {
+        return array_merge(parent::columns(), [
+            'name' => [
+                'attribute' => 'name',
+                'format' => 'raw',
+                'label' => Yii::t('hipanel:domain', 'Name'),
+                'filterAttribute' => 'name_ilike',
+                'value' => function (Zone $model) {
```

Comment:
```suggestion
                'value' => function (Zone $model): string {
```

Старайся не забывать